pkg/crc/machine/machine.go

```diff
@@ -114,18 +129,40 @@ func Start(startConfig StartConfig) (StartResult, error) {
 
 		vmState, err := host.Driver.GetState()
 		if err != nil {
-			logging.ErrorF("Error getting the state for host: %s", err)
+			logging.ErrorF("Error getting the state: %s", err)
 			result.Error = err.Error()
 		}
 
 		result.Status = vmState.String()
 	}
 
-	logging.InfoF(" Waiting 3m0s for the openshift cluster to be started ...")
-	time.Sleep(time.Minute * 3)
-	logging.InfoF(" To access the cluster as the system:admin user when using 'oc', run 'export KUBECONFIG=%s'", result.ClusterConfig.KubeConfig)
-	logging.InfoF(" Access the OpenShift web-console here: %s", result.ClusterConfig.ClusterAPI)
-	logging.InfoF(" Login to the console with user: kubeadmin, password: %s", result.ClusterConfig.KubeAdminPass)
+	// Post-VM start
+	host, err := libMachineAPIClient.Load(machineConfig.Name)
+	ip, err := host.Driver.GetIP()
+	if err != nil {
+		logging.ErrorF("Error getting the IP: %s", err)
+		result.Error = err.Error()
+	}
+
+	if driverInfo.UseDNSService {
+		servicePostStartConfig := services.ServicePostStartConfig{
+			Name: startConfig.Name,
+			IP:   ip,
+			// TODO: should be more finegrained
+			BundleMetadata: *crcBundleMetadata,
+			HostIP:         "localhost",
+		}
+		dns.RunPostStart(servicePostStartConfig)
+	}
+	//
+
+	if result.Error != "" {
```

Comment:
This should be `result.Error == "" ` since if there is error we are not going to tell user this information. or this can be as it as before.

Reply:
right... accidently committed this, as it allowed a faster startup and testing run ;-). Changed